tasks/install_targz.yml

```diff
@@ -15,12 +15,14 @@
     path: "{{ item.path }}"
     state: directory
     mode: "{{ item.mode | default('0755') }}"
+    owner: "{{ instance_user }}"
+    group: "{{ instance_group }}"
   loop:
     - {path: "{{ bin_dir }}{{ cartridge_app_name }}"}
     - {path: "{{ data_dir }}"}
     - {path: "{{ conf_dir }}"}
-    - {path: "{{ instance_run_path }}", mode: 777}
-    - {path: "{{ instance_systemd_path }}"}
+    - {path: "{{ instance_run_path }}"}
+    - {path: "{{ instance_work_dir }}"}
```

Comment:
Should it be ran for stateboard?
I remember that functions like a `get_instance_work_dir` accept `stateboard` flag and compute stateboard-specific directories, so there is no reason to run `Create stateboard folders` task.